pl_bolts/callbacks/vision/confused_logit.py

```diff
@@ -1,14 +1,17 @@
 import importlib
+from typing import Sequence, Tuple
 
 import torch
-from pytorch_lightning import Callback
-from torch import nn
+from pytorch_lightning import Callback, LightningModule, Trainer
+from torch import Tensor, nn
 
 from pl_bolts.utils.warnings import warn_missing_pkg
 
-_MATPLOTLIB_AVAILABLE = importlib.util.find_spec("matplotlib") is not None
+_MATPLOTLIB_AVAILABLE = importlib.util.find_spec("matplotlib") is not None  # type: ignore[attr-defined]
```

Comment:
Do we need `type: ignore` here? As I ran `mypy==0.790` without it, it raises no error...

Reply:
Yes, it ran well locally, but failed in https://github.com/PyTorchLightning/pytorch-lightning-bolts/runs/1585245200
not sure why

Reply:
Ok, I see. That's strange...